go/app/main.go

```diff
@@ -28,14 +39,63 @@ func root(c echo.Context) error {
 func addItem(c echo.Context) error {
 	// Get form data
 	name := c.FormValue("name")
+	category := c.FormValue("category")
 	c.Logger().Infof("Receive item: %s", name)
+	c.Logger().Infof("Receive item: %s", category)
 
 	message := fmt.Sprintf("item received: %s", name)
 	res := Response{Message: message}
 
+	// open json file & data
+	jsonFile, err := os.Open("items.json")
+	if err != nil {
+		log.Fatal("JSONファイルを開けません", err)
+		return err
+	}
+	defer jsonFile.Close()
+
+	jsonData, err := ioutil.ReadAll(jsonFile)
+	if err != nil {
+		log.Fatal("JSONデータを読み込めません", err)
+		return err
+	}
+	// convert json into go format
+	var items Items
+	json.Unmarshal(jsonData, &items)
```

Comment:
Don't forget to check the error returned by `json.Unmarshal` here.

Reply:
16d89552e86daf22217fa2b08d67ffb6eebd143a